uaclient/apt.py

```diff
@@ -455,14 +454,8 @@ def clean_apt_files(*, _entitlements=None):
             continue
         repo_file = ent_cls.repo_list_file_tmpl.format(name=ent_cls.name)
         pref_file = ent_cls.repo_pref_file_tmpl.format(name=ent_cls.name)
-
-        if os.path.exists(repo_file):
-            logging.info("Removing apt source file: %s", repo_file)
-            os.unlink(repo_file)
-
-        if os.path.exists(pref_file):
-            logging.info("Removing apt preferences file: %s", pref_file)
```

Comment:
Was there are reason for this specific logging message?

Reply:
Updated!